### ISSUE TITLE
The commshare menu doesn't allow sharing things to the AI when cheats are enabled

### ISSUE THREAD

**Issue by Sgathec** [ZeroK-RTS/Zero-K#1872]
I think it should support sharing to the AI as well if its a replacement for the menu that was used to share resources previously.

**Comment by mtroyka** [comment]
No. If cheats are enabled you can forcibly merge into an AI using /team x, but there are no AIs which currently support commsharing with players. The AI interface supports it but it's a pain to implement and can cause all kinds of issues and is entirely not worth it.

Otherwise the AI will spam commands at its units that will typically override the player's actions, and due to bandwidth optimizations a player's actions can easily break an AI or at least break down its coherent functionality.

You can still share units to an AI ally without using cheats, the same as with players, and AIs do handle that correctly in general since it's no different from capturing units with a domi.

**Comment by sprunk** [comment]
The point is not merging with AI. The point is giving units to AI. The new H widget does not allow that.

**Comment by mtroyka** [comment]
Ok so what does that have to do with cheats?

**Comment by Sgathec** [comment]
I would like to share units to the null AI for testing purposes.

**Comment by Sgathec** [comment]
Well, I don't know if allied AI players can be shared units to.

**Comment by Sgathec** [comment]
The point of the original post was that the new menu doesn't allow sharing units to an enemy null AI.

**Comment by sprunk** [comment]
That command does not give away an existing unit. It spawns a new one.

**Comment by mtroyka** [comment]
There has never been an interface that let you give existing units to an enemy, except maybe for random missions.

**Comment by sprunk** [comment]
Engine share dialog (old H menu) does that.
